dynawo/sources/Models/CPP/ModelNetwork/DYNModelBus.h

```diff
@@ -466,8 +469,7 @@ class ModelBus : public NetworkComponent::Impl {  ///< Generic AC network bus
   bool stateUmin_;  ///< whether U < Umin
 
   bool switchOff_;  ///< whether the bus was switched off
-  bool nodeFault_;  ///< whether a short-circuit is applied to the bus or not
-  // equivalent to switchOff_ but with discrete variable, to be able to switch off a node thanks to an outside event
+  // equivalent to switchOff but with discrete variable, to be able to switch off a node thanks to an outside event
```

Comment:
The second line could be deleted and was related to the deleted attribute

Reply:
This comment is for connectionState_ below